src/main/webapp/app/controllers/submission/submissionListController.js

```diff
@@ -539,21 +557,36 @@ vireo.controller("SubmissionListController", function (NgTableParams, $controlle
 
         var getValueFromArray = function (array, col) {
             var value = "";
-            for (var j in array) {
-                var member = array[j];
-                if (member.fieldPredicate !== undefined) {
-                    if (member.fieldPredicate.value === col.predicate) {
-                        value += value.length > 0 ? ", " + member.value : member.value;
-                    }
-                } else {
-                    var path = col.valuePath;
-                    var curr = member;
-                    for (var p = 1; p < path.length; p++) {
-                        curr = curr[path[p]];
+            var predicates = col.predicate.split(' ');
```

Comment:
Again, the same concern about splitting on the predicates that may have a single predicate that has a space in the name.